docs/reference/api/docker_remote_api_v1.25.md

```diff
@@ -3148,6 +3148,10 @@ Sets up an exec instance in a running container `id`
       "AttachStderr": true,
       "Cmd": ["sh"],
       "DetachKeys": "ctrl-p,ctrl-q",
+      "Env": [
+                   "FOO=bar",
+                   "BAZ=quux"
+             ],
```

Comment:
(nitpick), can you change the indenting to match other examples (we're not completely consistent yet, but trying to make it more consistent with each PR :smile:);

```patch
diff --git a/docs/reference/api/docker_remote_api_v1.25.md b/docs/reference/api/docker_remote_api_v1.25.md
index 5b29ef5..efd9114 100644
--- a/docs/reference/api/docker_remote_api_v1.25.md
+++ b/docs/reference/api/docker_remote_api_v1.25.md
@@ -3149,9 +3149,9 @@ Sets up an exec instance in a running container `id`
       "Cmd": ["sh"],
       "DetachKeys": "ctrl-p,ctrl-q",
       "Env": [
-                   "FOO=bar",
-                   "BAZ=quux"
-             ],
+        "FOO=bar",
+        "BAZ=quux"
+     ],
       "Privileged": true,
       "Tty": true,
       "User": "123:456"
```